client/verta/verta/registry/_validate_schema.py

```diff
@@ -104,7 +104,7 @@ def wrapper(self, input: Dict):
             ) from e
 
         # run function
-        output = f(self, input)
+        output = f(self, input, headers)
```

Comment:
This is going to break if the user didn't opt into headers for their model predict.

I think we could instead do

```python
def wrapper(self, input: Dict, *args, **kwargs):
    ...
    output = f(self, input, *args, **kwargs)
```
and that should work fine?

Reply:
I was wondering about this.